lazy-pages/src/tests.rs

```diff
@@ -76,12 +76,12 @@ fn read_write_flag_works() {
     }
 
     init_with_handler::<TestHandler>(
-        LazyPagesVersion::Version2,
+        LazyPagesVersion::Version1,
         vec![
             <WasmPage as PageU32Size>::size(),
             <GearPage as PageU32Size>::size(),
         ],
-        vec![LimitedStr::from_small_str("")],
+        vec![LimitedStr::from_small_str("gear_gas")],
```

Comment:
import global with name pls

Reply:
its only for tests. There is no way to use real wasm